ui/src/app/apps/app-details/app-details.component.ts

```diff
@@ -39,15 +43,28 @@ export class AppDetailsComponent implements OnInit {
       const appType: ApplicationType = params['appType'] as ApplicationType;
 
       console.log(`Retrieving app registration details for ${appName} (${appType}).`);
-      this.busy = this.appsService.getAppInfo(appType, appName).subscribe(data => {
+      const busy = this.appsService.getAppInfo(appType, appName)
```

Comment:
Also Spinner does not stay present when waiting for data to return from server for  the App Details .